bats/fb-katello-content.bats

```diff
@@ -116,6 +116,54 @@ setup() {
     --content-view="${CONTENT_VIEW}" --to-lifecycle-environment="${LIFECYCLE_ENVIRONMENT}" --from-lifecycle-environment="Library"
 }
 
+@test "export content view version" {
+  hammer content-export complete version --organization="${ORGANIZATION}" \
+    --content-view="${CONTENT_VIEW}" --version="1.0" | grep -q "Generated"
+}
+
+@test "create skeleton org, product, content view, and repo for import" {
+  hammer organization create --name="${IMPORT_ORG}" | grep -q "Organization created"
+  hammer product create --organization="${IMPORT_ORG}" --name="${PRODUCT}" | grep -q "Product created"
+  hammer content-view create --import-only --organization="${IMPORT_ORG}" \
+    --name="${CONTENT_VIEW}" | grep -q "Content view created"
+  hammer repository create --organization="${IMPORT_ORG}" \
+    --product="${PRODUCT}" --content-type="yum" --name "${YUM_REPOSITORY}" \
+    --url https://jlsherrill.fedorapeople.org/fake-repos/needed-errata/ | grep -q "Repository created"
+  repo_id=$(hammer --csv --no-headers repository list --organization="${IMPORT_ORG}" \
+  | grep ${YUM_REPOSITORY} | cut -d, -f1)
+  hammer content-view add-repository --organization="${IMPORT_ORG}" \
+  --name="${CONTENT_VIEW}" --repository-id=$repo_id | grep -q "The repository has been associated"
+}
+
+@test "import the exported content view" {
+  latest_export=$(hammer --output csv --no-headers content-export list --content-view "${CONTENT_VIEW}" --organization "${ORGANIZATION}"\
+   --content-view-version="1.0" --per-page=1 --order="id DESC")
+  # 16,,/var/lib/pulp/exports/Test_Organization/Test_CV/1.0/2020-12-11T16-04-08-00-00,Test CV 1.0,6,2020-12-11 16:04:12 UTC,2020-12-11 16:04:12 UTC
+  export_history_id=$(echo $latest_export | cut -d, -f1) # 16
+  export_path=$(echo $latest_export | cut -d, -f3)
+  # /var/lib/pulp/exports/Test_Organization/Test_CV/1.0/2020-12-11T16-04-08-00-00
+  import_path="/var/lib/pulp/imports/bats-test-$export_history_id"
+
+  mkdir -p $import_path
+  cp -r $export_path/* $import_path
+  if [ -f "${export_path}/metadata.json" ]; then
+    metadata_path="${export_path}/metadata.json"
+  else
+    metadata_path="$(pwd)/metadata-$export_history_id.json" # metadata-16.json
+  fi
+  # no grep here because hammer doesn't output any text on success
+  hammer content-import version --content-view="${CONTENT_VIEW}" --organization="${IMPORT_ORG}"\
+    --metadata-file=$metadata_path --path=$import_path
```

Comment:
If it looks like you are likely going to be using this above logic in multiple tests related to import, you might wanna refactor fetching the metadata_file part and put it in its own method
```bash
find_metadata_file() {
  local history_id=$1
  run hammer --output csv --no-headers  content-export list --id=$history_id --fields=path
  [ $status -eq 0 ]
  local path=${output}
  local  metadata_filename="metadata-$history_id.json" # metadata-16.json
  if [ -f "${path}/metadata.json" ]; then
    echo "${path}/metadata.json"
  else
    echo "$(pwd)/${metadata_filename}"
  fi
}
````

Reply:
I realized that `metadata-$export_history_id.json` will only exist if it can't write `metadata.json` to the original export path.  With proper permissions this will never happen, and hammer won't even need the `--metadata-file` parameter.  So I'm on the fence about whether to include this logic even here.

Reply:
Don't add if not needed :)